frame.go

```diff
@@ -167,6 +167,20 @@ func (f *Frame) SizedPayload() []byte {
 	return f.Payload[:f.Header.PayloadSize()]
 }
 
+// messageType returns the message type.
+func (f *Frame) messageType() messageType {
+	return f.Header.messageType
+}
+
+// Service returns the name of the destination service.
+func (f *Frame) Service() string {
```

Comment:
this was a quick hack I did, let's make it a little bit safer and easier to read by doing,
```go
const (
  serviceLenIndex = 1 /* flags */ + 4 /* ttl */ + 25 /* tracing */
  serviceIndex = serviceLenIndex + 1
)

if f.messageType() != messageTypeCallReq {
  panic("[...]")
}
serviceLen := f.Payload[serviceLenIndex])
return string(f.Payload[serviceIndex:serviceIndex + serviceLen])
```